protos/feast/core/FeatureView.proto

```diff
@@ -75,6 +75,9 @@ message FeatureViewSpec {
 
     // Whether these features should be served online or not
     bool online = 8;
+
+    // Needed for backwards compatible behaviour when fixing
```

Comment:
I think you need to update the new available id

Reply:
Also below

Reply:
removed this change